### ISSUE TITLE
Back button breaks "Update Project"

### ISSUE THREAD

**Issue by frafra** [ckan/ckan#6567]
**CKAN version**
2.9.4

**Describe the bug**
After a dataset is modified, if the user goes back, the "Update Project" button is disabled.

**Steps to reproduce**
1. Modify a dataset
2. Press Back (browser button)

**Expected behavior**
The back button should not break CKAN.

**Comment by amercader** [comment]
@frafra do you mean "Update dataset"?

**Comment by frafra** [comment]
@amercader yes, sorry, I fix that. I have a CKAN instance where datasets are renamed to projects :)

**Comment by Gauravp-NEC** [comment]
@frafra @amercader  , I have tried to reproduce this issue but the issue is not reproduced. Can you provide more details to reproduce this issue?

If dataset is updated and browser back button is pressed without clicking on `Update Dataset` button then definitely it will break the updates. Secondly, if dataset is updated and `Update Dataset` button is clicked to complete the dataset update task then CKAN will redirect to dataset page with updates in it.

**Comment by frafra** [comment]
> @frafra @amercader , I have tried to reproduce this issue but the issue is not reproduced. Can you provide more details to reproduce this issue?

Sure. Just click on `Update Dataset` and use the brower back button. When you click on `Update Dataset`, the button switch to inactive, so when the browser goes back, the DOM is restored as it was, and the button is still disabled.

**Comment by Gauravp-NEC** [comment]
@frafra , I have re-tried to reproduce this issue based on your above reply. Again, the issue is not reproduced. 
If I just click on `Update Dataset` button then the page is redirected back.

**Comment by frafra** [comment]
@Gauravp-NEC have you use the *browser* back button to go back to the metadata form? On Firefox, the latest DOM of the form page is shown, which is with the button disabled.

**Comment by frafra** [comment]
@Gauravp-NEC are you using Firefox with CKAN 2.9.5? I guess some browsers could reload the page entirely, when going back.

**Comment by Gauravp-NEC** [comment]
> @Gauravp-NEC are you using Firefox with CKAN 2.9.5? I guess some browsers could reload the page entirely, when going back.

@frafra , I am using Firefox with CKAN 2.9.4.

**Comment by hussnainwithss** [comment]
@frafra i have tried the steps you told and this is still not replicated.
